lib/echo/base.rb

```diff
@@ -37,10 +37,15 @@ def make_request(url, body)
 
       echo_response = Echo::Response.new(response)
       begin
-
-        Rails.logger.error "SOAP Response Error: #{echo_response.body.inspect}" if echo_response.error?
-
+        msg_hash = Hash.send('from_xml', echo_response.body)
+        if echo_response.error?
+          token_val = msg_hash.dig('Envelope', 'Body', 'Fault', 'detail','AuthorizationFault','Token')
+          msg_hash.dig('Envelope', 'Body', 'Fault', 'detail','AuthorizationFault').delete('Token') if token_val.present?
+        end
+        Rails.logger.error "SOAP Response Error: #{msg_hash.inspect}"
```

Comment:
Seems like this line will print regardless if there is an error or not. Is that what you really want here? "SOAP Resonse Error:..." will always be printed.

Reply:
Good catch. I'll fix this.